frontend/e-buddy/src/app/(demo)/bots/setup/page.tsx

```
@@ -1,153 +1,251 @@
                 </div>
-              </form>
-              <div className="mt-8 flex justify-end">
-                <button 
-                  onClick={nextStep} 
-                  className="bg-blue-600 text-white px-6 py-3 rounded-lg hover:bg-blue-700 transition-colors"
+              </div>
+              <div className="mt-10 flex justify-end">
+                <button
+                  onClick={nextStep}
+                  className="btn-primary flex items-center gap-2"
                 >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)